modules/Kconfig.renesas_fsp

```diff
@@ -48,3 +48,20 @@ config HAS_RENESAS_RA_RSIP_E51A
 	  Includes RSIP-E51A implementation for SCE driver
 
 endif
+
+config USE_RA_FSP_SCE
+	bool
+	help
+	  Enable RA FSP SCE driver
+
+config USE_RA_FSP_TIMER_ULPT
+	bool
+	help
+	  Enable RA FSP TIMER-ULPT driver
+
+config USE_RA_FSP_LPM
+	bool
+	default y
+	depends on PM
+	help
+	  Enable RA FSP LPM driver
```

Comment:
why does nothing in this file have an if or depends on?

Reply:
Thank you for checking, it's fix in https://github.com/zephyrproject-rtos/zephyr/pull/78208